rules/no-inline-units/no-inline-units.md

```diff
@@ -0,0 +1,13 @@
+# effector/no-inline-units
+
+Disallows to use inline units in methods
```

Comment:
Could you provide further information, why it is a bad idea to use inline units?

Reply:
ye sure